packages/i18nify-js/src/modules/types/geo.ts

```diff
@@ -1,248 +1,230 @@
+// extracted from https://www.henleyglobal.com/passport-index
+
 export type CountryCodeType =
-  | 'US'
-  | 'AQ'
-  | 'AG'
-  | 'AI'
-  | 'AS'
-  | 'BB'
-  | 'BM'
-  | 'BS'
-  | 'CA'
-  | 'DM'
-  | 'DO'
-  | 'GD'
-  | 'GU'
-  | 'JM'
-  | 'KN'
-  | 'KY'
-  | 'LC'
-  | 'MP'
-  | 'MS'
-  | 'PR'
-  | 'SX'
-  | 'TC'
-  | 'TT'
-  | 'VC'
-  | 'VG'
-  | 'VI'
-  | 'RU'
-  | 'KZ'
-  | 'EG'
-  | 'ZA'
-  | 'GR'
-  | 'NL'
-  | 'BE'
-  | 'FR'
-  | 'ES'
-  | 'HU'
-  | 'IT'
-  | 'VA'
-  | 'RO'
-  | 'CH'
-  | 'AT'
-  | 'GB'
-  | 'GG'
-  | 'IM'
-  | 'JE'
-  | 'DK'
-  | 'SE'
-  | 'NO'
-  | 'SJ'
-  | 'PL'
-  | 'DE'
-  | 'PE'
-  | 'MX'
-  | 'CU'
-  | 'AR'
-  | 'BR'
-  | 'CL'
-  | 'CO'
-  | 'VE'
-  | 'MY'
-  | 'AU'
-  | 'CC'
-  | 'CX'
-  | 'ID'
-  | 'PH'
-  | 'NZ'
-  | 'SG'
-  | 'TH'
-  | 'JP'
-  | 'KR'
-  | 'VN'
-  | 'CN'
-  | 'TR'
-  | 'IN'
-  | 'PK'
-  | 'AF'
-  | 'LK'
-  | 'MM'
-  | 'IR'
-  | 'SS'
-  | 'MA'
-  | 'EH'
-  | 'DZ'
-  | 'TN'
-  | 'LY'
-  | 'GM'
-  | 'SN'
-  | 'MR'
-  | 'ML'
-  | 'GN'
-  | 'CI'
-  | 'BF'
-  | 'NE'
-  | 'TG'
-  | 'BJ'
-  | 'MU'
-  | 'LR'
-  | 'SL'
-  | 'GH'
-  | 'NG'
-  | 'TD'
-  | 'CF'
-  | 'CM'
-  | 'CV'
-  | 'ST'
-  | 'GQ'
-  | 'GA'
-  | 'CG'
-  | 'CD'
-  | 'AO'
-  | 'GW'
-  | 'IO'
-  | 'AC'
-  | 'SC'
-  | 'SD'
-  | 'RW'
-  | 'ET'
-  | 'SO'
-  | 'DJ'
-  | 'KE'
-  | 'TZ'
-  | 'UG'
-  | 'BI'
-  | 'MZ'
-  | 'ZM'
-  | 'MG'
-  | 'RE'
-  | 'YT'
-  | 'ZW'
-  | 'NA'
-  | 'MW'
-  | 'LS'
-  | 'BW'
-  | 'SZ'
-  | 'KM'
-  | 'SH'
-  | 'TA'
-  | 'ER'
-  | 'AW'
-  | 'FO'
-  | 'GL'
-  | 'GI'
-  | 'PT'
-  | 'LU'
-  | 'IE'
-  | 'IS'
-  | 'AL'
-  | 'MT'
-  | 'CY'
-  | 'FI'
-  | 'AX'
-  | 'BG'
-  | 'LT'
-  | 'LV'
-  | 'EE'
-  | 'MD'
-  | 'AM'
-  | 'BY'
-  | 'AD'
-  | 'MC'
-  | 'SM'
-  | 'UA'
-  | 'RS'
-  | 'ME'
-  | 'XK'
-  | 'HR'
-  | 'SI'
-  | 'BA'
-  | 'MK'
-  | 'CZ'
-  | 'SK'
-  | 'LI'
-  | 'FK'
-  | 'BZ'
-  | 'GT'
-  | 'SV'
-  | 'HN'
-  | 'NI'
-  | 'CR'
-  | 'PA'
-  | 'PM'
-  | 'HT'
-  | 'GP'
-  | 'BL'
-  | 'MF'
-  | 'BO'
-  | 'GY'
-  | 'EC'
-  | 'GF'
-  | 'PY'
-  | 'MQ'
-  | 'SR'
-  | 'UY'
-  | 'CW'
-  | 'BQ'
-  | 'TL'
-  | 'NF'
-  | 'BN'
-  | 'NR'
-  | 'PG'
-  | 'TO'
-  | 'SB'
-  | 'VU'
-  | 'FJ'
-  | 'PW'
-  | 'WF'
-  | 'CK'
-  | 'NU'
-  | 'WS'
-  | 'KI'
-  | 'NC'
-  | 'TV'
-  | 'PF'
-  | 'TK'
-  | 'FM'
-  | 'MH'
-  | 'KP'
-  | 'HK'
-  | 'MO'
-  | 'KH'
-  | 'LA'
-  | 'BD'
-  | 'TW'
-  | 'MV'
-  | 'LB'
-  | 'JO'
-  | 'SY'
-  | 'IQ'
-  | 'KW'
-  | 'SA'
-  | 'YE'
-  | 'OM'
-  | 'PS'
-  | 'AE'
-  | 'IL'
-  | 'BH'
-  | 'QA'
-  | 'BT'
-  | 'MN'
-  | 'NP'
-  | 'TJ'
-  | 'TM'
-  | 'AZ'
-  | 'GE'
-  | 'KG'
-  | 'UZ'
-  | 'EU';
+  | 'AF' // Afghanistan
```

Comment:
How did the countryCodes reduce ?

Reply:
A couple of them were uninhabited islands which are not even recognized by Passport Index so removed those plus some were merged into one country so modified those as well.